Content.Server/Pointing/EntitySystems/PointingSystem.cs

```diff
@@ -96,11 +96,17 @@ public bool InRange(EntityUid pointer, EntityCoordinates coordinates)
             }
         }
 
-        public bool TryPoint(ICommonSession? session, EntityCoordinates coords, EntityUid pointed)
+        public bool TryPoint(ICommonSession session, EntityCoordinates coords, EntityUid pointed)
         {
-            var mapCoords = coords.ToMap(EntityManager);
-            if ((session as IPlayerSession)?.ContentData()?.Mind?.CurrentEntity is not { } player)
+            if (session.AttachedEntity is not { } player)
```

Comment:
Does this work properly for ghosts? This is the commit that originally changed it to be what it was: https://github.com/ElectroJr/space-station-14/commit/c042b462202ea51a5204ee1067604eb8a0510534

Reply:
It seems to work? And if this doesn't, then we have a lot of other problems because just about all interaction logic assumes that the attached entity is the "current entity" trying to perform an interaction.